hack/verify-crds-version-upgrade.sh

```diff
@@ -1,102 +0,0 @@
-#!/bin/bash
-
-# This script is to verify whether api spec is kept same when switching from crd v1beta1 to v1.
-# The process is:
-# 1. start a kube-apiserver.
-# 2. apply all the v1beta1 crds from the last commit, and dump the spec of each crd to #crdName-before file.
-# 3. apply all the v1 crds from the current head, and dump the spec of each crd to #crdName-after file.
-# 4. compare #crdName-before and #crdName-after for v1beta1 crd if it is switched to v1 in dev branch.
-
-source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
-
-TMP_ROOT="${SCRIPT_ROOT}/_tmp"
-KUBECTL="./_output/tools/kubebuilder/kubectl --server=http://127.0.0.1:8080"
-mkdir -p "${TMP_ROOT}"
-
-if [ ! -f ./_output/tools/bin/yq ]; then
-    mkdir -p ./_output/tools/bin
-    curl -s -f -L https://github.com/mikefarah/yq/releases/download/2.4.0/yq_$(go env GOHOSTOS)_$(go env GOHOSTARCH) -o ./_output/tools/bin/yq
```

Comment:
I'm very happy to see Curl commands removed from build scripts.  (We still have one in `hack/verify-crds.sh`, but maybe one day we'll remove that Curl command too.)

Reply:
I'm working on this on a local branch, trying to get rid of downloading and installing any binaries in favour of just running them using `go run`